### PR TITLE
 Bump @vocdoni/sdk package from v0.0.16 to v0.4.0 

### PR DESCRIPTION
**Pull request by andreslucena**
I've also added a couple commands that are useful to regenerate and view the database, and an output for the SDK version in the JS console

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,6 @@ group :development, :test do
 end
 
 group :development do
-  gem "i18n-tasks", "~> 0.9.37"
```
Comment (andreslucena) [inline Gemfile:23]: @microstudi just to be sure, you're removing this as its not necessary, because its already a dependency of `decidim-dev`, right?

Comment (microstudi) [reply Gemfile:23]: yes

**Review by microstudi — APPROVED** [review]

**Review by microstudi — CHANGES_REQUESTED** [review]
we need to check the tests!

**Comment by andreslucena** [comment]
@microstudi last time I talked with the people from Vocdoni, they told me that the failing spec is related to how they handle their versions and APIs while they're on development. Basically, for now: 

v0.4.0 - works with the `dev` environment
v0.3.x - works with the `stag` environment

In a few days/weeks they'll migrate `stag` to be compatible with the v0.4.0 API. Until that time we have a couple options: 

1. To change our environment to `dev` 
2. To upgrade to v0.3.x instead of v0.4.0
3. To wait a couple days/weeks until `stag` is compatible with v0.4.0 

As we don't have any current development in our side on the near future, I think it's better for now to just wait (3rd option) and this start blocking us then we can discuss the other options. 

For the moment I sent a rerun to see if the `stag`/0.4.0 thing already happened 🤞🏽

**Comment by andreslucena** [comment]
Closing this as it was superseded by #72

**Pull request closed without merging**